### PR TITLE
Add 4.0.1 release post

### PR DESCRIPTION
**Pull request by rafaelfranca**

### CODE REVIEW

File: _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+```
+
+We chose to revert the behavior because it added a major backward incompatibility that made
+harder to have an upgrade path without major changes in the application code. Also we consider
+the older behavior a bug since it behaves differently from all the others scope methods when they
+are chained. So we took the most conservative path of reverting it to be consistent with the idea
+of having a smoother upgrade path to Rails 4.
+
+For those who want the old behavior you can use the [`.reorder`](http://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-reorder)
+or [`.unscope`](http://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-unscope)
```
Comment (robin850) [inline _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown:41]: "you can use the ... **methods**" or remove "the", what do you think ?

File: _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+includes more than 460 commits.
+
+This release comes up with an important change on how Active Record handles subsequent `order` calls.
+In Rails 4.0.0 when you do something like this:
+
+```ruby
+User.order("name asc").order("created_at desc")
+```
+
+The later called `order` will be prepended in the ORDER BY clause resulting on this SQL:
```
Comment (robin850) [inline _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown:21]: I'm not sure, I may be wrong but "The last call to `order` will be ...", no ?

Comment (rafaelfranca) [inline _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown:21]: latter called is also right

File: _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+To see the full list of changes, [check out all the commits on
+GitHub](https://github.com/rails/rails/compare/v4.0.0...v4.0.1).
+
+## SHA-1
+
+If you'd like to verify that your gem is the same as the one I've uploaded,
+please use these SHA-1 hashes:
+
+* 482e0c05fb25ae9a8d261f301054fc182e7b4447  actionmailer-4.0.1.rc1.gem
```
Comment (sikachu) [inline _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown]: Don't forget to change these! (both SHA and version)

File: _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown (hunk 2 above)
Comment (robin850) [inline _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown:21]: s/later/latter then, no ? :grin:

Comment (rafaelfranca) [inline _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown:21]: Right

File: _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown (hunk 3 above)
Comment (rafaelfranca) [inline _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown]: Done

File: _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown [hunk 4]
```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: '[ANN] Rails 4.0.1 has been released!'
+category: releases
+author: rafaelfranca
+published: false
```
Comment (sikachu) [inline _posts/2013-11-01-Rails-4-0-1-has-been-released.markdown]: Don't forget to change this! :P

**Pull request merged**